packages/retail-ui-extensions/src/components/index.ts

```diff
@@ -8,3 +8,4 @@ export * from './SegmentedControl';
 export * from './Tag';
 export * from './Dialog';
 export * from './SearchBar';
+export * from './Types';
```

Comment:
Nit, and no reason to change right now: we typically would not use a pascal-case file name unless it contains an actual component